XMonad/Layout/IndependentScreens.hs

```diff
@@ -137,6 +204,11 @@ marshallPP :: ScreenId -> PP -> PP
 marshallPP s pp = pp { ppRename = ppRename pp . unmarshallW
                      , ppSort = fmap (marshallSort s) (ppSort pp) }
 
+-- | @marshallPP'@ uses @marshallSort'@ instead of marshallSort, the rest is the same with marshallPP.
```

Comment:
Would be lovely to have some sort of explanation why one might want that, perhaps an example…